backend/src/routes/__tests__/user.spec.js

```diff
@@ -62,22 +64,36 @@ describe('Register flow', () => {
       })
       .catch(done);
   });
+}),
 
-  it('GET /users/token/:id should return 200 and the user token', (done) => {
+describe('Get email token of user', () => {
+  it('GET /users/email-token/:id as ADMIN should return 200 and the user token', (done) => {
     request(app)
-      .get(`/users/token/${userId}`)
+      .get(`/users/email-token/${userId}`)
       .set('Authorization', `Bearer ${adminToken}`)
       .expect(200)
       .expect('Content-Type', /json/)
       .then((res) => {
-        expect(res.body.token).toBeDefined();
-        userEmailToken = res.body.token;
+        expect(res.body.emailToken).toBeDefined();
+        userEmailToken = res.body.emailToken;
         done();
       })
       .catch(done);
   });
```

Comment:
Ha oui ok, je comprends pourquoi tu as fais ça.

Il vaudrait mieux que tu import le service du user dans le fichier de test et que tu appelle la method `getEmailToken` plutot que de faire une route qui ne sert qu'au tests.

Et n'oublie pas de check is le mail token est bien `undefined` dans les autres tests